rc/scdoc.kak

```diff
@@ -0,0 +1,54 @@
+# https://git.sr.ht/~sircmpwn/scdoc
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.]scd %{
+    set-option buffer filetype scdoc
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=scdoc %{
+    require-module scdoc
+    hook -once -always window WinSetOption filetype=.* %{ remove-hooks window scdoc-.+ }
+}
+
+hook -group scdoc-highlight global WinSetOption filetype=scdoc %{
+    add-highlighter window/scdoc ref scdoc
+    hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/scdoc}
+}
+
+
+provide-module scdoc %{
+
+# Highlighters
+# ‾‾‾‾‾‾‾‾‾‾‾‾
+
+add-highlighter shared/scdoc regions
+add-highlighter shared/scdoc/inline default-region regions
+add-highlighter shared/scdoc/inline/text default-region group
```

Comment:
Wouldn't it be more logical to have `scdoc` as a regular `group`, under which all the regex below would be declared, and a single regions group containing `codeblock`?

Reply:
Maybe, I just copied that from markdown.kak.

Reply:
Will you have time to make those modifications? I can tweak your code with my own commits, after merge, if not.

In any case, could you please amend your commit so that the title follows the convention I use? It should be `rc: Highlight SCDOC`.